src/deploy/functions/runtimes/node/versioning.ts

```diff
@@ -45,6 +46,7 @@ export function getFunctionsSDKVersion(sourceDir: string): string | void {
     const child = spawn.sync("npm", ["list", "firebase-functions", "--json=true"], {
       cwd: sourceDir,
       encoding: "utf8",
+      timeout: NPM_COMMAND_TIMEOUT_MILLIES,
```

Comment:
We actually removed this spawn in a recent commit, do you mind pulling in the changes from master?

Reply:
@colerogers thank you very much, rebased from master